config/main.py

```diff
@@ -621,6 +621,9 @@ def config():
 
 config.add_command(aaa.aaa)
 config.add_command(aaa.tacacs)
+
+import kube
```

Comment:
Change to `from . import kube`. This should fix the LGTM alerts.

Also add a comment like you did in show/main.py ("# Import kube commands")

Reply:
Alerts fixed.